recipes/install.rb

```diff
@@ -0,0 +1,25 @@
+#
+# Cookbook Name:: joerd
+# Recipe:: install
+#
+
+template "#{node[:joerd][:cfg_path]}/#{node[:joerd][:cfg_file]}" do
+  source 'joerd-config.yaml.erb'
+end
+
+template "#{node[:joerd][:cfg_path]}/#{node[:joerd][:logging_file]}" do
+  source 'joerd-logging.conf.erb'
+end
+
+# pip install base requirements
+file node[:joerd][:pip_requirements_location] do
+  content node[:joerd][:pip_requirements].join("\n")
+end
+
+execute 'pip_install' do
+  command "pip install -U -r #{node[:joerd][:pip_requirements_location]}"
+end
+
+execute 'pip_install_joerd' do
+  command "pip install #{node[:joerd][:pip_joerd_repo]}"
```

Comment:
Alternatively, we could just add the version of joerd we want to install to the requirements list too and install in one command.

Actually, that might be preferable, because repeated installs will not alter the joerd install, because the joerd requirement is already satisfied.